packages/components/src/components/table/table-row.tsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import classNames from 'classnames';
-import PropTypes from 'prop-types';
 
-const Row = ({ children, className, has_hover }) => {
+type TRow = {
+    children: React.ReactNode;
+    className?: string;
+    has_hover?: boolean;
+};
+
+const Row = ({ children, className, has_hover }: TRow) => {
```

Comment:
```suggestion
const Row = ({ children, className, has_hover }: React.PropsWithChildren<TRow>) => {
```
```
type TRow = {
    className?: string;
    has_hover?: boolean;
};
```
@yauheni-kryzhyk-deriv please also address an issue in packages/p2p/src/components/orders/order-table/order-table-row.jsx where we pass an additional `style` prop to this component but we don't use it inside it. 🙂

Reply:
same is here: children are required